quiz/web/src/modules/genericTests/LanguageSwitcher/LanguageSwitcher.tsx

```diff
@@ -0,0 +1,29 @@
+import { useTranslation } from "react-i18next";
+
+const langs = {
+    en: { nativeName: "English" },
+    de: { nativeName: "Deutsch" },
+    pl: { nativeName: "Polish" },
+};
+
+function LanguageSwitcher() {
+    const { i18n } = useTranslation();
+
+    return (
+        <div>
+            {Object.keys(langs).map((lang) => (
```

Comment:
```
src/modules/genericTests/LanguageSwitcher/LanguageSwitcher.tsx:22:22 - error TS7053: Element implicitly has an 'any' type because expression of type 'string' can't be used to index type '{ en: { nativeName: string; }; de: { nativeName: string; }; pl: { nativeName: string; }; }'.
  No index signature with a parameter of type 'string' was found on type '{ en: { nativeName: string; }; de: { nativeName: string; }; pl: { nativeName: string; }; }'.
```

Reply:
`Object.keys(langs).map((lang: keyof typeof langs) => (`